Dockerfile

```
@@ -59,9 +61,11 @@ RUN set -ex; \
     rm -rf /tmp/pear /var/cache/apk/*
 # DO NOT FORGET TO CHECK THE LANGUAGE PACK DOWNLOAD URL BELOW
 # DO NOT FORGET TO UPDATE "image-version" FILE
-ENV OSTICKET_VERSION=1.16.3 \
-    OSTICKET_SHA256SUM=7d9c1ae7663e66d189b70d29fe02844d5ec2324a0f4cca545205a55f29ff72aa
-RUN set -ex; \
+ENV OSTICKET_VERSION=1.17.2 \
+    OSTICKET_SHA256SUM=53aa6349c0ee6367d4370cc663a8047d3038f5e0d3668f42b3f90f20534fb717
+RUN --mount=type=bind,source=utils/verify-plugin.php,target=/var/www/html/verify-plugin.php \
```

```suggestion
RUN --mount=type=bind,source=utils/verify-plugin.php,target=/tmp/verify-plugin.php,readonly \
```
IMO it's better to not mount things into the application directory unless absolutely necessary. God knows what kind of side effects this can have on an app like this one.

---

Done in 796297e41a245ae8910fa9bfa3f1a38b9bd2f795